docs/general/ad-filtering/create-own-filters.md

```diff
@@ -116,21 +120,21 @@ Cosmetic rules are based on using a special language named CSS, which every brow
 #### Trusted filters {#trusted-filters}
 
 Some rules can be used only in trusted filters. This category includes:
-> * filter lists [created by the AdGuard team](../adguard-filters),
-> * custom filter lists installed as `trusted`,
-> * user rules.
+* filter lists [created by the AdGuard team](../adguard-filters),
+* custom filter lists installed as `trusted`,
+* user rules.
 
 ## Basic rules
 
 The most simple rules are so-called *Basic rules*. They are used to block requests to specific URLs. Or to unblock it, if there is a special marker "@@" at the beginning of the rule. The basic principle for this type of rules is quite simple: you have to specify the address and additional parameters that limit or expand the rule scope.
 
-> **Sub-requests**
->
-> Basic rules for blocking requests are applied only to **sub-requests**. That means they will not block the loading of the page.
+**Sub-requests**
+
+Basic rules for blocking requests are applied only to **sub-requests**. That means they will not block the loading of the page.
```

Comment:
Basic rules for blocking requests are applied only to **sub-requests**. That means they will not block the loading of the page unless it is explicitly specified with a `$document` modifier.